release_notes/ocp-4-16-release-notes.adoc

```diff
@@ -198,11 +201,11 @@ The Network Observability Operator releases updates independently from the {prod
 
 [id="workload-partitioning-enhancement"]
 ==== Workload partitioning enhancement
-With this release, platform pods deployed with a workload annotation that includes both CPU limits and CPU requests will have the CPU limits accurately calculated and applied as a CPU quota for the specific pod. In prior releases, if a workload partitioned pod had both CPU limits and requests set, they were ignored by the webhook. The pod did not benefit from workload partitioning and was not locked down to specific cores. This update ensures the requests and limits are now interpreted correctly by the webhook. 
+With this release, platform pods deployed with a workload annotation that includes both CPU limits and CPU requests will have the CPU limits accurately calculated and applied as a CPU quota for the specific pod. In earlier releases, if a workload partitioned pod had both CPU limits and requests set, they were ignored by the webhook. The pod did not benefit from workload partitioning and was not locked down to specific cores. This update ensures the requests and limits are now interpreted correctly by the webhook.
```

Comment:
```suggestion
With this release, platform pods, deployed with a workload annotation that includes both CPU limits and CPU requests, have the CPU limits accurately calculated and applied as a CPU quota for the specific pod. In earlier releases, if a workload partitioned pod had both CPU limits and requests set, they were ignored by the webhook. The pod did not benefit from workload partitioning and was not locked down to specific cores. This update ensures the requests and limits are now interpreted correctly by the webhook.
```

Reply:
IBMSG suggests to avoid future tense. Also, I think you need some commas around this phrase to make the sentence meaning a bit clearer.